.github/workflows/release-branch.yaml

```diff
@@ -1,75 +1,136 @@
-
-name: Release Branch Creation
+# This creates a new release branch from the main branch.
+# It serves as the cutoff point for the next minor release.
+# From this point onward only bug fixes and critical changes will be accepted onto the release
+# branch as backports from main. At the same time, the main branch will be open for new features
+# and changes for the next minor release.
+name: Release Branch Cutoff
 
 on:
   workflow_dispatch:
-    inputs:
-      tag:
-        type: string
-        description: "Tag name (if other than execution base)"
-        required: false
-        default: ""
+
+permissions:
+  # Necessary to write the branch
+  contents: write
 
 jobs:
-  check-and-create:
+  cutoff-preconditions:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+      repository-projects: read
+    outputs:
+      minor: ${{ steps.get-minor.outputs.minor }}
+      branch: ${{ steps.verify-branch.outputs.branch }}
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v2
+        with:
+          app_id: ${{ secrets.OCMBOT_APP_ID }}
+          private_key: ${{ secrets.OCMBOT_PRIV_KEY }}
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          fetch-depth: 0
+          token: ${{ steps.generate_token.outputs.token }}
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: '${{ github.workspace }}/go.mod'
+          cache: false
+      - name: Get Minor
+        id: get-minor
+        run: |
+          set -e
+          minor="$(go run ./api/version/generate print-major-minor)"
+          echo "minor=$minor" >> $GITHUB_OUTPUT
+          echo "Current Major-Minor Version: $minor"
+      - name: Verify Branch does not exist
+        id: verify-branch
+        run: |
+          set -e
+          minor="v${{ steps.get-minor.outputs.minor }}"
+          branch="releases/$minor"
```

Comment:
is it releases/x.y.z or release/x.y.z ?

Reply:
it is `releases` (plural). See https://github.com/open-component-model/ocm/branches/all?query=releases